include/OpenSpaceToolkit/Astrodynamics/Trajectory.hpp

```diff
@@ -165,6 +172,56 @@ class Trajectory
     /// @return Static trajectory
     static Trajectory Position(const physics::coordinate::Position& aPosition);
 
+    /// @brief Constructs a trajectory for a given strip, specified ground speed and start instant
+    ///
+    /// @code{.cpp}
+    ///             LLA startLLA = { 0.0, 0.0, 0.0 };
+    ///             LLA endLLA = { 1.0, 0.0, 0.0 };
+    ///             Real groundSpeed = 1000.0;
+    ///             Instant startInstant = Instant::DateTime(DateTime::Parse("2020-01-01 00:00:00"), Scale::UTC);
+    ///             Earth earth = Earth::WGS84();
+    ///             Trajectory trajectory = Trajectory::GroundStrip(startLLA, endLLA, groundSpeed, startInstant, earth);
+    /// @endcode
+    ///
+    /// @param aStartLLA A start LLA
+    /// @param anEndLLA An end LLA
+    /// @param aGroundSpeed A ground speed
+    /// @param aStartInstant A start instant
+    /// @param aCelestial Celestial body
+    /// @return GroundStrip trajectory
+    static Trajectory GroundStrip(
+        const LLA& aStartLLA,
+        const LLA& anEndLLA,
+        const Real& aGroundSpeed,
+        const Instant& aStartInstant,
+        const Celestial& aCelestial = Earth::WGS84()
```

Comment:
Nit: since we have now the concept of central body (as well as an Environment singleton), maybe the default value could be Environment.getInstance().getCentralBody() ?